src/meta/meta_bulk_load_service.cpp

```diff
@@ -17,12 +17,23 @@
 
 #include <dsn/dist/fmt_logging.h>
 #include <dsn/utility/fail_point.h>
+#include <dsn/utility/flags.h>
 
 #include "meta_bulk_load_service.h"
 
 namespace dsn {
 namespace replication {
 
+DSN_DEFINE_uint32("meta_server",
+                  bulk_load_max_failover_count,
```

Comment:
```suggestion
                  bulk_load_max_rollback_times,
```

I prefer to call it retry_times or rollback times. Because in your description you call it "rollback", it's inconsistent to name it "failover".